lib/config.rb

```diff
@@ -45,8 +45,8 @@ class Config
       'retries' => 10
     }
 
-    def self.find_config
-      Rockbot.resolve_relative 'rockbot.json'
+    def self.get_relative(path)
```

Comment:
yeah i didnt even spot that. an improvement for the future could be taking arguments with names i.e. ./rockbot.rb -c config.json

Reply:
That was how I originally intended to implement this change, but upon consideration, I don't know what other CLI options this program even needs since it already has a config file.